subcommand/acl-controller/command.go

```diff
@@ -29,6 +32,10 @@ const (
 	// Binding rules don't support a '/' character, so we need compatible IAM role tag names.
 	authMethodServiceNameTag = "consul.hashicorp.com.service-name"
 	authMethodNamespaceTag   = "consul.hashicorp.com.namespace"
+
+	// anonTokenID is the well-known ID for the anonymous ACL token.
+	anonTokenID    = "00000000-0000-0000-0000-000000000002"
+	anonPolicyName = "anonymous-token-policy"
```

Comment:
This is the same name [consul-k8s uses](https://github.com/hashicorp/consul-k8s/blob/aa05d3db2158e0887f49a0f5354893603ff6b6f2/control-plane/subcommand/server-acl-init/anonymous_token.go#L9-L43) 🤔 

Thinking about how this works if we have both and ECS an EKS dc/partition. Both sides would attempt to create and attach this policy, but that should be fine. Only one policy would be created, and both configure the policy with the same rules.

Reply:
Yeah I intentionally used the same policy name as consul-k8s.. and I think this is OK because both consul-k8s and consul-ecs check if the policy already exists before attempting to create it. I think that the only time that both sides would attempt to create and attach the policy is in a race condition where they both read a non-existent policy at the same time, which should be rare.